src/main/java/org/zalando/nakadi/domain/PaginationLinks.java

```diff
@@ -0,0 +1,46 @@
+package org.zalando.nakadi.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.Optional;
+
+@Immutable
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
+public class PaginationLinks {
+
+    private final Optional<Link> prev;
+
+    private final Optional<Link> next;
+
+    public PaginationLinks() {
+        this(Optional.empty(), Optional.empty());
+    }
+
+    public PaginationLinks(final Optional<Link> prev, final Optional<Link> next) {
```

Comment:
What do you think about using LInk in constructor but wrap it in Optional inside constructor or mark links as nullable? I think it will help use to support our 'null policy' because right now we can pass null in constructor.